process/elasticproc/check.go

```diff
@@ -39,6 +41,9 @@ func checkArguments(arguments *ArgElasticProcessor) error {
 	if check.IfNilReflect(arguments.OperationsProc) {
 		return elasticIndexer.ErrNilOperationsHandler
 	}
+	if check.IfNilReflect(arguments.IndexTokensHandler) {
```

Comment:
Perhaps we should rename this to something like `CrossChainTokenPropertiesHandler` or smth similar

Reply:
Added in TODO `MX-16499` for refactoring